pytest/tests/mocknet/local_test_node.py

```diff
@@ -229,6 +229,26 @@ def run_cmd(cmd):
         )
 
 
+def make_snapshot(self, snapshot_id):
```

Comment:
Where are these called? Looks like they should be methods of `LocalTestNeardRunner` or something.